pom.xml

```diff
@@ -24,8 +24,8 @@
   </properties>
 
   <modules>
-    <module>client</module>
-    <module>web</module>
+    <module>src/client</module>
```

Comment:
why we modify the project structure?

Reply:
To keep aligned with other todo templates, they all place the application code part in a `src` folder, e.g. 
https://github.com/Azure-Samples/todo-java-mongo/tree/main/src
https://github.com/Azure-Samples/todo-java-mongo-aca/tree/main/src

Reply:
I don't think we need to keep aligned with other todo templates, keep the original pattern looks good. How do you think about it @stliu ?

Reply:
and for the [pet clinic sample](https://github.com/azure-samples/spring-petclinic-microservices), we also need to decide that whether to move all source code modules to an isolated folder.